Dockerfile

```diff
@@ -0,0 +1,52 @@
+# hadolint global ignore=DL3008
+FROM debian:buster
```

Comment:
Why debian rather than ubuntu?

Reply:
Most docker official images (eg: python, memcached, etc) use Debian or Alpine for their base images.

Debian buster still supports python 2.7 (default python version). I am planning to test using the system packaged version of `gunicorn`.
Need to go all the way back to Ubuntu 18.04 for a system packaged version of `gunicorn` with python 2.7 support.

Debian buster is supported until [June 2024](https://wiki.debian.org/LTS)
Ubuntu 18.04 is supported until [April 2023](https://wiki.ubuntu.com/Releases)